skimage/metrics/simple_metrics.py

```diff
@@ -164,6 +164,7 @@ def peak_signal_noise_ratio(image_true, image_test, *, data_range=None):
     image_true, image_test = _as_floats(image_true, image_test)
 
     err = mean_squared_error(image_true, image_test)
+    data_range = float(data_range)  # prevent overflow for small integer types
     return 10 * np.log10((data_range**2) / err)
```

Comment:
```suggestion
    return 20 * np.log10(data_range / err)
```

Reply:
Took me a second. Great catch! :grinning:

Reply:
Hups, we entirely missed that there's a division in there. So pulling out the **2 should actually be
```python
    return 20 * np.log10(data_range / np.sqrt(err))
```
I'll revert to the previous version, since all variations seem similar in complexity.

Reply:
oh man, my math is rusty. thanks! for the good catch.